### DISCUSSION TITLE
[Unstable] Round ending too fast when falling unconscious

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14119] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In multiplayer, if a player is the last (or only) player alive, and they fall unconscious, the round will immediately end. Since they technically never died, they are not considered "dead" for the purposes of an Ironman run, and so they can continue with the save even if everyone was unconscious.

Additionally, the round ends even if the player could have been healed (either by a bot or some other effect like meds or matriarch genes) back to consciousness. Which is slightly annoying if the injuries could have been healed, but the server takes everyone back to the lobby anyway. It would be better if there was a timer that started when everyone was unconscious, or just didn't go back to lobby until everyone is properly dead.

### Reproduction steps

1. Launch a multiplayer campaign and be the only player
2. get a medic bot to follow you and give sufficient meds such that they could heal you
3. give yourself just enough damage to knock you unconscious
4. Observe that the server takes you back to the lobby before you can be healed

And for the other issue
1. Launch an ironman multiplayer campaign and be the only player
2. Spawn a bunch of enemies near you and be attacked
3. Observe that the server takes you back to the lobby before you can die, and so you can reload the ironman save

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this and found no issues
Closing discussion as resolved
